### PR TITLE
Consistent language about list-based headers

### LINKED ISSUE #289: Audit: how we talk about list field values
Similar to #193, we need to use consistent language when talking about list values.

Includes:

- [ ] Vary: #286

### PR DESCRIPTION
**Pull request by mnot**
Does not address Vary (see #286), and some more discussion of some of the conditional request headers might be warranted.

Fixes #289.

### CODE REVIEW

File: draft-ietf-httpbis-semantics-latest.xml [hunk 1]
```diff
@@ -2414,9 +2415,9 @@ Host: www.example.org
...
 </t>
 <t>
-   A sender &MAY; generate comments in the Via header field to identify the
+   A sender &MAY; generate comments to identify the
    software of each recipient, analogous to the <x:ref>User-Agent</x:ref> and
-   <x:ref>Server</x:ref> header fields. However, all comments in the Via field
+   <x:ref>Server</x:ref> header fields. However, comments in Via
    are optional, and a recipient &MAY; remove them prior to forwarding the
    message.
 </t>
```
Comment (reschke) [inline draft-ietf-httpbis-semantics-latest.xml]: appears not really related...

File: draft-ietf-httpbis-semantics-latest.xml [hunk 2]
```diff
@@ -1400,14 +1400,15 @@ deployed and not likely to be registered in a timely manner otherwise.</t>
    as quoted-string (<xref target="quoted.strings"/>).
 </t>
 <t>
-   Because commas (",") are used as a generic delimiter between field values,
-   they need to be treated with care if they are allowed in the field value.
-   Typically, components that might contain a comma are protected with
-   double-quotes using the quoted-string ABNF production.
+   Because commas (",") are used as a generic delimiter between members of a
+   list-based field value, they need to be treated with care if they are
+   allowed in those members. Typically, list members that might contain a
```
Comment (royfielding) [inline draft-ietf-httpbis-semantics-latest.xml]: Maybe "allowed as data within those members"

File: draft-ietf-httpbis-semantics-latest.xml [hunk 3]
```diff
@@ -1400,14 +1400,15 @@ deployed and not likely to be registered in a timely manner otherwise.</t>
...
+   comma are protected with double-quotes using the quoted-string ABNF
```
Comment (royfielding) [inline draft-ietf-httpbis-semantics-latest.xml]: "are protected with double-quotes using the quoted-string ABNF production" -> "are enclosed in a quoted-string"

File: draft-ietf-httpbis-semantics-latest.xml [hunk 4]
```diff
@@ -1571,7 +1572,7 @@ deployed and not likely to be registered in a timely manner otherwise.</t>
 </t>
 <section title="Sender Requirements">
 <t>
-   In any production that uses the list construct, a sender &MUST-NOT;
+   In any production that uses this construct, a sender &MUST-NOT;
```
Comment (royfielding) [inline draft-ietf-httpbis-semantics-latest.xml]: I preferred is as "uses the list construct"

**Pull request merged**